### PR TITLE
Document 'threads' option

### PR DESCRIPTION
**Pull request by tschorr**
Add documentation about `threads` option to README.

### CODE REVIEW

**Comment by tschorr** [comment]
@jenkins-plone-org please run jobs

File: news/126.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Add documentation for `threads` option.
```
Comment (jensens) [inline news/126.bugfix]: this file should be named ``126.doc``, see https://towncrier.readthedocs.io/en/actual-freaking-docs/quickstart.html#creating-news-fragments

Comment (tschorr) [reply news/126.bugfix]: done in https://github.com/plone/plone.recipe.zope2instance/pull/126/commits/8b387a95cef792da96a80c3d154b15dff687338e

**Comment by tschorr** [comment]
@jenkins-plone-org please run jobs

**Comment by tschorr** [comment]
Jenkins failure is unrelated, there is a full disk:

```
Traceback (most recent call last):
  File "bin/test", line 420, in <module>
    '--test-path', '/home/jenkins/.buildout/eggs/cp27m/Products.validation-2.1.1-py2.7.egg',
  File "/home/jenkins/.buildout/eggs/cp27m/collective.xmltestreport-2.0.1-py2.7.egg/collective/xmltestreport/runner.py", line 66, in run
    failed = run_internal(defaults, args, script_parts=script_parts)
  File "/home/jenkins/.buildout/eggs/cp27m/collective.xmltestreport-2.0.1-py2.7.egg/collective/xmltestreport/runner.py", line 83, in run_internal
    runner.options.output.writeXMLReports()
  File "/home/jenkins/.buildout/eggs/cp27m/collective.xmltestreport-2.0.1-py2.7.egg/collective/xmltestreport/formatter.py", line 297, in writeXMLReports
    outputFile.close()
IOError: [Errno 28] No space left on device
```

**Pull request merged**